Be more honest about the mapping between app and transport (#188)
ietf-wg-masque/draft-ietf-masque-h3-datagram: draft-ietf-masque-h3-datagram.md

HTTP connection. While HTTP datagrams are associated with HTTP requests, they
are not part of message content; instead, they are intended for use by HTTP
extensions (such as the CONNECT method), and are compatible with all versions of
HTTP.

> When HTTP is based on top of a transport protocol that supports unreliable

Reviewer: ```suggestion
When HTTP is running over a transport protocol that supports unreliable
```
